heartbeat/pgsql

```diff
@@ -861,6 +865,61 @@ pgsql_stop() {
 }
 
 #
+# reload configuration (or restart if changes cannot be applied without reload)
+#
+pgsql_reload() {
+    local rc
+    local tmp_file
+    tmp_file="${OCF_RESKEY_tmpdir}/recovery.conf.check"
+
+    ocf_log info "pgsql RA reload called"
+
+    # check if we can just reload
+    # or restart needed (in case changed parameters in recovery.conf)
+
+    # check if we a master
+    pgsql_real_monitor
+    rc=$?
+    if [ $rc -eq $OCF_RUNNING_MASTER ]; then
+        ocf_log debug "we are a master -> reload is enough"
+        exec_with_retry 0 reload_conf
+        return $?
+    fi
+
+    make_recovery_conf "${tmp_file}"
+    rc=$?
+    if [ $rc -eq 0 ]; then
+        #runasowner "diff --new-file ${RECOVERY_CONF} ${tmp_file} >/dev/null"
+        if [ -f ${RECOVERY_CONF} ]; then
+            ocf_log debug "comparing recovery.conf"
+            runasowner "cmp --silent ${RECOVERY_CONF} ${tmp_file}"
+            rc=$?
+        else
+            # recovery.conf missing
+            if [ -s ${tmp_file} ]; then
+                ocf_log debug "new recovery.conf"
+                rc=1
+            else
+                ocf_log debug "empty recovery.conf"
+                rc=0
+            fi
+        fi
+        ocf_log debug "removing temporary file"
+        runasowner "rm -f ${tmp_file}"
+        if [ $rc -eq 0 ]; then
+            ocf_log debug "recovery.conf is unchanged -> reload is enough"
```

Comment:
I do not understand the intention here: reloading the config. when the config. is unchanged??
'reload_conf' is supposed to be used to reflect the changes in recovery.conf.